docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -77,7 +77,9 @@
     description:
       Change multiple applications either by upgrading existing ones or creating new ones.
       If there is an update to an already running application, the application gets upgraded.
-      All instances of this application get replaced by the new version.
+      Instances of this application will be started or restarted
+      according to the usual logic to meet the requirements of the new definition. Usually, restart is required
+      if the application configuration was changed (except for scaling changes).
```

Comment:
has changed maybe?

Reply:
Maybe it would be more clear if we did a new sentence here? (except for scaling changes) -> Restart is not necessary for scaling changes.